src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private final int SIZE_OF_ARRAY = 12;
+    private final int[] numbers = new int[SIZE_OF_ARRAY];
 
-    public int total = -1;
+    private final int RETURN_ERROR = -1;
+    public int total = RETURN_ERROR;
```

Comment:
wartość „total” i stała RETURN_ERROR są różnymi wartościami. To musi być osobne wartosci